docs/VnVPlan/VnVPlan.tex

```diff
@@ -167,11 +167,71 @@ \subsection{SRS Verification Plan}
 
 \subsection{Design Verification Plan}
 
-\wss{Plans for design verification}
+The purpose of this design verification plan is to outline the specific methods, roles, and tools that will be used to ensure the design accurately implements the project requirements and adheres to best practices for scalability, security, and maintainability.
+
+\subsubsection{Team Member Roles}
```

Comment:
iffy about these roles and description

Reply:
might want to add our project overseer as a user to verify things work out from a user perspective

Reply:
we should follow test driven, so each developer should be responsible for their own test cases when they create a new feature.